modular/uploader/upload_task.go

```diff
@@ -55,6 +58,18 @@ func (u *UploadModular) PreUploadObject(ctx context.Context, uploadObjectTask co
 		log.CtxErrorw(ctx, "failed to pre upload object, object not create")
 		return ErrNotCreatedState
 	}
+
+	taskState, _, errGetUploadObjectState := u.baseApp.GfSpClient().GetUploadObjectState(ctx, uploadObjectTask.GetObjectInfo().Id.Uint64())
```

Comment:
Move to 87 line after u.baseApp.GfSpClient().CreateUploadObject is better.
Corner case:
1. old upload is uploading;
3. new upload check is ok; 
3. old upload is uploaded;
4. new upload overwrites old upload.